pkg/services/gitopswriter/gitopswriter.go

```diff
@@ -206,6 +277,9 @@ func removeKustomizeResources(app models.Application, repoDir, clusterName strin
 	}, nil
 }
 
+func getUserDirRepoPath(clusterName string) string {
+	return filepath.Join(git.WegoRoot, git.WegoClusterDir, clusterName, "user")
+}
 func getUserKustomizationRepoPath(clusterName string) string {
```

Comment:
With the path being pretty complicated, we should update this to call the getUserDirRepoPath() instead of rebuilding the path here.